.vscode/launch.template.json

```diff
@@ -72,8 +72,12 @@
     },
     {
       "id": "testName",
-      "type": "promptString",
-      "description": "Enter the test name to run, leave empty to run all"
+      "type": "command",
+      "command": "extension.commandvariable.promptStringRemember",
```

Comment:
We probably need to update the doc here https://github.com/ChainSafe/lodestar/blob/d85e73aa1273591bd7211987e6196427e6c75d14/docs/pages/contribution/tools/debugging.md?plain=1#L7

to remind users to install the extension

Reply:
completely forgot we had a section in docs about VS code debugging, good idea to add it there